Library/Homebrew/cmd/tap-info.rb

```diff
@@ -31,9 +30,9 @@ def tap_info
     end
 
     if ARGV.json == "v1"
-      print_tap_json(taps)
+      print_tap_json(taps.sort_by {|tap| tap.to_s})
```

Comment:
Run `brew style` and it'll point out a couple of style tweaks for this line (and the change below, too).

Reply:
Ah I see, thanks.